pkg/daemon/update.go

```diff
@@ -1741,7 +1741,10 @@ func (dn *Daemon) atomicallyWriteSSHKey(keys string) error {
 	}
 
 	var authKeyPath string
-	if dn.os.IsFCOS() {
+	if dn.os.IsFCOS() || dn.os.IsSCOS() {
```

Comment:
I've just realized here that this is https://issues.redhat.com/browse/COS-1589 and this is not yet done.